x-pack/plugins/cloud_security_posture/public/components/translations.ts

```diff
@@ -40,28 +40,27 @@ export const CSP_EVALUATION_BADGE_PASSED = i18n.translate(
   }
 );
 
-export const NO_DATA_CONFIG_TITLE = i18n.translate('xpack.csp.pageTemplate.noDataConfigTitle', {
-  defaultMessage: 'Understand your cloud security posture',
-});
-
-export const NO_DATA_CONFIG_SOLUTION_NAME = i18n.translate(
-  'xpack.csp.pageTemplate.noDataConfig.solutionNameLabel',
-  {
+export const PACKAGE_NOT_INSTALLED_TEXT = {
+  PAGE_TITLE: i18n.translate('xpack.csp.cspPageTemplate.packageNotInstalled.pageTitle', {
+    defaultMessage: 'Integration package not installed',
```

Comment:
Please revert this text change, it was on purpose for this specific use case where our integration is not yet installed

Reply:
I've consulted with @amar-rama-elastic, he suggested using `Install Integration to get started` as the title. I think it makes better sense since we now actually check for the installation status and also the line right below is a default "learn more about x" which is set by the `KibanaPageTempalte['noDataConfig']` so it's probably better to be a bit more accurate when displaying the problem and not repeat ourselves twice.

Of course, if I'm still missing the reasoning for the previous text please feel free to reject/comment.